### ISSUE TITLE
Legacy voices

### ISSUE THREAD

**Issue by HamletEagle** [elevenlabs/elevenlabs-python#325]
Hello,

Is it possible to continue using legacy voices from the Python API? For example, I was using "Josh", but I noticed it is no longer available in the API. `client.voices.get_all()` doesn't return it anymore.

I looked on the website and saw that the name got changed to "Josh (Legacy)", so I tried adding "(Legacy)" in the Python API without any success.

Thanks!

**Comment by HamletEagle** [comment]
Thank you! Here is how it can be done in the python API:

```
from elevenlabs.core import RequestOptions
request_options = RequestOptions(additional_query_parameters={"show_legacy": "true"})
audio = client.generate(text="Hello, World!", voice='Josh', model="eleven_multilingual_v2", request_options=request_options)
```

**Comment by dsinghvi** [comment]
@HamletEagle @ahmedebid  this is released as part of `v1.6.0rc0`, please let us now if you have any issues
